packages/components/src/angle-picker-control/README.md

```diff
@@ -0,0 +1,40 @@
+# AnglePickerControl
+
+AnglePickerControl is a React component to render a UI that allows users to pick an angle.
+Users can choose an angle in a visual UI with the mouse by dragging an angle indicator inside a circle or by directly inserting the desired angle in a text field.
+
+## Usage
+
+
+```jsx
+import { useState } from '@wordpress/element';
+import { AnglePickerControl } from '@wordpress/components';
+
+const MyAnglePicker = () => {
+	const [ angle, setAngle ] = useState();
+	return <AnglePickerControl value={ angle } onChange={ setAngle } />;
+};
+```
+
+## Props
+
+The component accepts the following props.
+
+### label
+
+Label to use for the angle picker. If not set the a translated label "Angle" is used.
+
+- Type: `String`
+- Required: No
+
+### value
+The current value of the input.
+
+- Type: `String | Number`
```

Comment:
Why is it a string or number? It's not clear what the expected value should be? 0 to 360?

Reply:
Hi @youknowriad, I documented as a String or Number because passing a value as a string or number would work well, also if the value is not between 0 and 360 things would also work well as mathematically these values are possible (although they could be represented as something between 0 and 360). But on second thought even though things work it does not mean they should be used that way so I updated type to be a number and said value should be between 0 and 360.